docs(walrus): add known gotcha callouts (#3426)
MystenLabs/walrus: docs/content/http-api/storing-blobs.mdx

keywords: [ walrus, HTTP, API, publisher, store, blob, PUT, deletable, permanent, quilt ]
---

:::caution No public Mainnet publisher

> Walrus has no public unauthenticated publisher on Mainnet, and there will not be one — a publisher pays WAL for every blob it stores. On Mainnet, run your own authenticated publisher (or use the [Upload Relay](/docs/operator-guide/upload-relay) or TypeScript SDK directly). The public publisher endpoints below are for Testnet, where WAL has no value.

Reviewer: ```suggestion
Walrus has no public unauthenticated publisher on Mainnet. There are no plans to create one. On Mainnet, run your own authenticated publisher (or use the [Upload Relay](/docs/operator-guide/upload-relay) or TypeScript SDK directly). The public publisher endpoints below are for Testnet, where WAL has no monetary value.
```

Reviewer: also add link to the TS SDK docs
